main.tf

```diff
@@ -37,8 +37,8 @@ resource "azurerm_linux_web_app" "main" {
 
   site_config {
     application_stack {
-      docker_image     = "appsvcsample/python-helloworld"
-      docker_image_tag = "latest"
+      docker_image     = "talexandru87/todo-app:latest"
```

Comment:
Your existing app service uses `talexandru87/todo-app:prod` - is this intentionally different?